functions.php

```diff
@@ -898,6 +898,8 @@ function twentytwenty_get_customizer_css( $type = 'front-end' ) {
 		// Get variables
 		$accent 		= get_theme_mod( 'twentytwenty_accent_color' );
 		$accent_default = '#CD2653';
+		$text 		= get_theme_mod( 'twentytwenty_text_color' );
```

Comment:
Tabs should not be used for non-indentation spacing.

>**Rule of thumb:** Tabs should be used at the beginning of the line for indentation, while spaces can be used mid-line for alignment.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation

Reply:
I run the file through https://github.com/WordPress/WordPress-Coding-Standards. Guess something went sideways. I'll look it up again.